src/iotjs_binding.h

```diff
@@ -216,4 +219,28 @@ jerry_value_t iotjs_jhelper_eval(const char* name, size_t name_len,
 
 jerry_value_t vm_exec_stop_callback(void* user_p);
 
+/**
+ * Dynamic module defintions (.iotjs)
+ */
+#define IOTJS_CURRENT_MODULE_VERSION() ((uint32_t)0x00000001)
```

Comment:
The () after the macro name is a bit strange to me. And we could use simple decimal numbers, increasing a hex number is brain killing.